addons/point_of_sale/views/pos_assets_index.xml

```diff
@@ -31,6 +31,7 @@
                 '__session_info__': session_info,
                 'login_number': login_number,
                 'pos_session_id': pos_session_id,
+                'nomenclature_id': nomenclature_id,
```

Comment:
Not a fan of stuffing more things into the odoo global. Maybe we can just add it into the __session_info__ and acces it through `@web/session`?